package.json

```diff
@@ -112,7 +112,8 @@
     "build:scripts": "babel src --out-dir dist/js --ignore .test.js,__mocks__",
     "build:less": "mkdir -p dist/less && cp -r less/* dist/less",
     "build:sass": "mkdir -p dist/sass && cp -r sass/patternfly-react/* dist/sass && node-sass --output-style compressed --include-path sass --include-path $npm_package_sassIncludes_patternfly --include-path $npm_package_sassIncludes_bootstrap --include-path $npm_package_sassIncludes_fontAwesome -o dist/css sass/patternfly-react.scss",
-    "lint": "eslint --rulesdir lint-rules/ --fix --max-warnings 0 src storybook && yarn stylelint",
+    "lint": "eslint --rulesdir lint-rules/ --max-warnings 0 src storybook && yarn stylelint",
+    "lint-fix": "eslint --rulesdir lint-rules/ --fix --max-warnings 0 src storybook && yarn stylelint",
```

Comment:
optional:  move the eslint cmd/args to something like `lint:js` and then call that will --fix:
```
"lint": "yarn lint:js && lint:styles",
"lint:js": "eslint --rulesdir lint-rules/ --max-warnings 0 src storybook",
"lint:styles": "yarn stylelint"
"lint-fix": "yarn lint:js --fix && yarn lint:styles --fix",
```

Reply:
That's much better, thanks!